runtime/android/core_internal/src/org/xwalk/core/internal/XWalkWebContentsDelegateAdapter.java

```diff
@@ -91,6 +91,12 @@ public boolean addMessageToConsole(int level, String message, int lineNumber,
     }
 
     @Override
+    public void showRepostFormWarningDialog() {
+        if (mXWalkContentsClient == null) return;
```

Comment:
`if (mXWalkContentsClient != null) mXWalkContentsClient.resetSwipeRefreshHandler();` is better?